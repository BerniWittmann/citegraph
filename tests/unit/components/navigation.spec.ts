import { shallowMount, RouterLinkStub } from '@vue/test-utils'
import { i18n } from '../setupPlugins'

import Navigation from '@/components/Navigation.vue'

describe('components/Navigation.vue', () => {
  it('renders', () => {
    const wrapper = shallowMount(Navigation, {
      i18n,
      mocks: {
        $route: {
          name: 'home'
        }
      },
      stubs: {
        RouterLink: RouterLinkStub
      }
    })
    expect(wrapper.html()).toMatchSnapshot()
  })
})
