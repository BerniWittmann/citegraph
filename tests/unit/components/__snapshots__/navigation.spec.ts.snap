// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/Navigation.vue renders 1`] = `
<el-menu-stub mode="horizontal" defaultactive="home" router="true" menutrigger="hover" activetextcolor="#0E79B2" collapsetransition="true" class="nav"><a class="nav-image-wrap">
    <div class="nav-image"></div>
  </a></el-menu-stub>
`;
